source/threads.tex

```diff
@@ -11121,7 +11121,7 @@
 Once evaluation of \tcode{invoke(std::move(g), std::move(xyz))} begins, the function is no longer
 considered deferred.
 
-\recommended
+ \recommended
```

Comment:
Is this trick of adding a space before the macro documented anywhere as the correct way to use one without starting a new paragraph?

Reply:
No.

I've changed my check script so that it complains about `\recommended` (and similar introducers) without `\pnum` inside `\itemdecl` only.  In the (rather exotic) case we have here, a `\pnum` isn't correct, because we're inside an `\itemize` environment. The space in front silences the check script. Maybe I should augment it even more so that stuff inside `\itemize` is ignored.